python/tank/util/yaml_cache.py

```diff
@@ -256,7 +256,7 @@ def _populate_cache_item_data(self, item):
         """
         path = item.path
         try:
-            with open(path, "r") as fh:
+            with open(path, "r", encoding='utf8') as fh:
```

Comment:
Maybe the linter warning was caused because of the different types of quoting used here. Let's use double quoting for strings as the rest of the file, please.